examples/mistral/mistral-dpo.yml

```diff
@@ -0,0 +1,103 @@
+#Note that we are switching from the regular chat template to chatml.
+#If you experience problems with the special tokens, training for more epochs can help.
+#After trainig, merge the model before inference otherwise you might
+#face problems with the special tokens.
+
+base_model: mistralai/Mistral-7B-Instruct-v0.2
+model_type: MistralForCausalLM
+tokenizer_type: LlamaTokenizer
+
+load_in_8bit: false
+load_in_4bit: true
+strict: false
+
+chat_template: chatml
+rl: dpo
+datasets:
+  - path: olivermolenschot/alpaca_messages_dpo_test
+    type: chat_template.default
+    field_messages: conversation
+    field_chosen: chosen
+    field_rejected: rejected
+    message_field_role: role
+    message_field_content: content
+    roles:
+      system:
+        - system
+      user:
+        - user
+      assistant:
+        - assistant
+
+dataset_prepared_path:
+val_set_size: 0.05
+output_dir: ./outputs/dpo-lora-out-5
+
+sequence_len: 64
+sample_packing: false
+pad_to_sequence_len: true
+
+adapter: lora
```

Comment:
I notice you used `load_in_4bit: True` but did `adapter: lora`. Did you perhaps mean `qlora` or to use `load_in_8bit: True`?

Reply:
I meant to minimize the model size for testing. But I can push the example as a `qlora` and `load_in_8bit: True` so that it can be easier for people using this example as later on.

Reply:
Actually is the `load_in_8bit` for the model or the lora adapter? and if it is for the lora adapter how is setting qlora achieving anything different?

Reply:
Hey! We would use `load_in_4bit: True` with `adapter: qlora` and 8bit with lora. 

This is for the model quantized, not the adapter. If you use lora/qlora, could you perhaps rename the file to add `-lora.yml` or `-qlora.yml` at the end, so it's easier to distinguish?

Reply:
Done!